### PR TITLE
Reference Tut Uppercase

### PR DESCRIPTION
**Pull request by ChristopherDavenport**
I believe this is the necessary fix. Doesn't build on `1.2.0` to confirm. Investigating that presently.

### CODE REVIEW

**Comment by tpolecat** [comment]
This looks good. Have we verified that it fixes the issue reported on Gitter?

**Comment by ChristopherDavenport** [comment]
I have tested locally and this resolves the issue.

File: modules/plugin/src/main/scala/tut/TutPlugin.scala [hunk 1]
```diff
@@ -16,11 +16,12 @@ object TutPlugin extends AutoPlugin {
   override def trigger = noTrigger
 
   override def requires = sbt.plugins.JvmPlugin
+  override def projectConfigurations = Seq(autoImport.Tut)
```
Comment (olafurpg) [inline modules/plugin/src/main/scala/tut/TutPlugin.scala:19]: I can confirm that adding `.configs(Tut)` to my tut-enabled project (equivalent of `def projectConfigurations` here) removes the warning in my builds 👍

**Comment by tpolecat** [comment]
Great, thanks! I'll try to get a release out today.

**Pull request merged**